packages/runtime-core/__tests__/apiApp.spec.ts

```diff
@@ -148,7 +151,7 @@ describe('api: createApp', () => {
           a: 1
         }
       },
-      created(this: any) {
```

Comment:
This usage of `any` here is fine because type information doesn't provide actual benefits here - instead, it makes the rest of the test more verbose. This change feels typing for typing's sake to me.